scientist-softserv/webstore
# 363 request specs

# Story
stubbed request cypress e2e tests are passing **locally**. they will still fail in ci due to configuration. (ref: https://github.com/scientist-softserv/webstore/issues/378)

- ref: #363

# Screenshots / Video
<img width="951" alt="image" src="https://github.com/scientist-softserv/webstore/assets/29032869/5df1bb7a-427b-4fee-847a-d3af1b2c05d0">

# Testing
1. pull this branch
2. ensure you have all of the environment variables in the `env` property of "cypress.config.js" set up.
3. run `yarn cypress:e2e`
4. choose the "request" test
